source/PvL_E.rst

```diff
@@ -55,85 +55,77 @@ error before the colon, and specifics about the error after the colon.
+executed after the ``try`` one, if no exception occurred. A ``finally`` block
+is executed in any case. With all the statements, a ``try`` clause looks like:
 
     .. sourcecode:: python3
         :linenos:
-        
-        def exists(filename):
-            try:
-                f = open(filename)
-                f.close()
-                return True 
-            except:
-                return False 
 
-    .. The try statement in this function was already introduced previously
-       (the same code), so I thought it would be appropriate to add an else
-       clause here.
-       
-    .. admonition:: A template to test if a file exists, without using exceptions
-
-        The function we've just shown is not one we'd recommend. It opens
-        and closes the file, which is semantically different from asking "does
-        it exist?". How?  Firstly, it might update some timestamps on the file.
-        Secondly, it might tell us that there is no such file if some other
-        program already happens to have the file open, or if our permission
-        settings don't allow us to open the file.
-
-        Python provides a module called ``os.path`` within the ``os`` module. It
-        provides a number of useful functions to work with paths, files and directories,
-        so you should check out the help.  
-        
-            .. sourcecode:: python3
-                :linenos:
-            
-                import os
-                
-                # This is the preferred way to check if a file exists.
-                if os.path.isfile("c:/temp/testdata.txt"):
-                   ...
-           
-   
-            
-We can use multiple ``except`` clauses to handle different kinds of
-exceptions (see the
-`Errors and Exceptions <http://docs.python.org/py3k/tutorial/errors.html>`__
-lesson from Python creator Guido van Rossum's
-`Python Tutorial <http://docs.python.org/py3k/tutorial/index.html>`__
-for a more complete discussion of exceptions).  So the program could do
-one thing if the file does not exist, but do something else if the file
-was in use by another program.
+        user_input = input('Type a number:')
+        try:
+            # Try do do something that could fail.
+            user_input_as_number = float(user_input)
+        except ValueError:
+            # This will be executed if a ``ValueError`` is raised.
+            print('You did not enter a number.')
+        else:
+            # This will be executed if not exception got raised in the
+            # ``try`` statement.
+            print('The square of your number is ', user_input_as_number**2)
+        finally:
+            # This will be executed whether or not an exception is raised.
+            print('Thank you')
+
+When using a ``try`` clause, you should have as little as possible in the
+``try`` block. If too many things happen in that block, you take the risk
+handle an unexpected exception.
```

Comment:
... you take the risk *of handling* an unexpected exception
or
... you risk handling an unexpected exception